account_payment_partner/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 
 {
     "name": "Account Payment Partner",
-    "version": "13.0.1.1.3",
+    "version": "13.0.1.1.4",
```

Comment:
Why do you change this manually?

Reply:
I have not changed it, it must be an automatic process.